src/composables/useCurrentUser.js

```diff
@@ -16,6 +16,53 @@ export function useCurrentUser() {
 		return pkp.currentUser.id;
 	}
 
+	function getCurrentUserName() {
+		return pkp.currentUser.username;
+	}
+
+	function getCurrentUserFullName() {
+		return pkp.currentUser.fullName;
+	}
+
+	function getCurrentUserInitials() {
+		return pkp.currentUser.initials;
+	}
+
+	function getUserLoggedInAsUserName() {
+		if (!isUserLoggedInAs()) {
+			return null;
+		}
+
+		const userLoggedInAs = getUserLoggedInAs();
+		return userLoggedInAs?.username || '';
```

Comment:
This should cover the `null` return

```suggestion
		const userLoggedInAs = getUserLoggedInAs();
		return userLoggedInAs?.username || '';
```

Alternatively (in case the `if` is required for some reason...), I think you can assume the user is accessible


```suggestion
		if (!isUserLoggedInAs()) {
			return null;
		}

		const userLoggedInAs = getUserLoggedInAs();
		return userLoggedInAs.username;
```